terms/demographics/CDR.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "sage.annotations-demographics.CDR-0.0.1",
+  "type": "number",
+  "description": "Clinical Dementia Rating",
+  "anyOf": [
+    {
+      "const": 0,
+      "description": "None"
+    },
+    {
+      "const": 0.5,
+      "description": "Very mild"
+    },
+    {
+      "const": 1,
+      "description": "Mild"
+    },
+    {
+      "const": 2,
+      "description": "Moderate"
+    },
+    {
+      "const": 3,
+      "description": "Severe"
+    }
+  ]
+}
```

Comment:
These will need a source for each. I suggest: https://doi.org/10.1017/S1041610297004870.

Reply:
> These will need a source for each. I suggest: https://doi.org/10.1017/S1041610297004870.

Agree. I believe this is the source Ben recomended